src/test/java/bao/task/TaskTest.java

```diff
@@ -0,0 +1,137 @@
+package bao.task;
+
+import bao.main.Bao;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TaskTest {
+    public static final String TEST_FILE_PATH = "data/test_task.json.txt";
+    private Bao bao;
+
+    /**
+     * Sets up a test file with known content.
+     */
+    @BeforeEach
+    public void setUp() {
+        Bao bao = new Bao(TEST_FILE_PATH);
+    }
+
+    /**
+     * Tests if todo task is created correctly using fromString method.
+     */
+    @Test
+    public void createToDoTaskUsingFromStringTest() {
```

Comment:
perhaps for testing methods, instead of camel case, this convention can be used --unitBeingTested_descriptionOfTestInputs_expectedOutcome ?